.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,26 @@
+on: [push]
+
+jobs:
+  maintainance:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - shell: pwsh
+      run: "& ${{ github.workspace }}/CommonLibSF/cmake/make-directives.ps1 ${{ github.workspace }}/CommonLibSF"
+
+    - uses: DoozyX/clang-format-lint-action@v0.16.2
+      with:
+        source: '.'
+        exclude: './docs'
+        extensions: 'c,cc,cpp,cxx,h,hpp,hxx,inl,inc,ixx'
+        clangFormatVersion: 16
+        inplace: True
+
+    - uses: EndBug/add-and-commit@v9
+      with:
+        author_name: maintenance
+        message: 'chore: maintenance'
```

Comment:
Technically a lint would be `style:` vs `chore:`.

Reply:
In CS we're using style: 🎨 apply clang-format change

Reply:
Good to know, I set it to chore because it also re-generates the `Starfield.h` so it could handle future PR's better.